### ISSUE TITLE
Remove dependency on gulp-clean as per Gulp docs

### ISSUE THREAD

**Issue by peterblazejewicz** [thejameskyle/marionette-wires#31]
When running `npm install` I'm getting this warning:
```
marionette-wires git:(master) npm install
npm WARN deprecated gulp-clean@0.3.1: use gulp-rimraf instead
```
Can build task be updated to use Gulp recommended `del` based solution?
https://github.com/gulpjs/gulp/blob/master/docs/recipes/delete-files-folder.md

I've rewritten a task and it works ok:
```diff
commit 60dbdea35d6c1083571ae0bdbd4086911c1d57d7
Author: Peter Blazejewicz <peter.blazejewicz@gmail.com>
Date:   Sat Jan 24 20:26:09 2015 +0100

    Use del in place of gulp-clean
    
    As per the Gulp docs:
    https://github.com/gulpjs/gulp/blob/master/docs/recipes/delete-files-fol
    der.md

diff --git a/gulpfile.js b/gulpfile.js
index 497d483..4344167 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -1,6 +1,7 @@
 var gulp = require('gulp');
 var $ = require('gulp-load-plugins')();
 var browserify = require('browserify');
+var del = require('del');
 var watchify = require('watchify');
 var source = require('vinyl-source-stream');
 var stylish = require('jshint-stylish');
@@ -12,10 +13,10 @@ var reload = browserSync.reload;
 
 var api = require('./api/api');
 
-gulp.task('clean', function() {
-  return gulp.src('app/tmp', {read: false})
-    .pipe($.plumber())
-    .pipe($.clean());
+gulp.task('clean', function(cb) {
+  del([
+    'app/tmp'
+  ], cb);
 });
 
 gulp.task('html', function() {
diff --git a/package.json b/package.json
index ab124fa..d754266 100644
--- a/package.json
+++ b/package.json
@@ -33,10 +33,10 @@
     "browserify-shim": "^3.8.2",
     "chai": "^1.10.0",
     "chai-jquery": "^2.0.0",
+    "del": "^1.1.1",
     "express": "^4.11.0",
     "gulp": "^3.8.10",
     "gulp-autoprefixer": "^2.1.0",
-    "gulp-clean": "^0.3.1",
     "gulp-env": "^0.2.0",
     "gulp-jshint": "^1.9.0",
     "gulp-less": "^2.0.1",
```

**Comment by thejameskyle** [comment]
Sounds good, wanna make a pr?

**Comment by peterblazejewicz** [comment]
@thejameskyle done:
https://github.com/thejameskyle/marionette-wires/pull/33
Thanks!

**Comment by thejameskyle** [comment]
Fixed in #33
